[2024-12-12T08:45:01.077+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-12-12T08:45:01.099+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dbt_orchestrator.test_connection.dbt_test_connection scheduled__2024-12-12T08:30:00+00:00 [queued]>
[2024-12-12T08:45:01.107+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dbt_orchestrator.test_connection.dbt_test_connection scheduled__2024-12-12T08:30:00+00:00 [queued]>
[2024-12-12T08:45:01.107+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-12-12T08:45:01.117+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): test_connection.dbt_test_connection> on 2024-12-12 08:30:00+00:00
[2024-12-12T08:45:01.124+0000] {standard_task_runner.py:63} INFO - Started process 1308 to run task
[2024-12-12T08:45:01.126+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dbt_orchestrator', 'test_connection.dbt_test_connection', 'scheduled__2024-12-12T08:30:00+00:00', '--job-id', '290', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp3x49xlba']
[2024-12-12T08:45:01.128+0000] {standard_task_runner.py:91} INFO - Job 290: Subtask test_connection.dbt_test_connection
[2024-12-12T08:45:01.170+0000] {task_command.py:426} INFO - Running <TaskInstance: dbt_orchestrator.test_connection.dbt_test_connection scheduled__2024-12-12T08:30:00+00:00 [running]> on host 34fdc836a0bb
[2024-12-12T08:45:01.251+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ma7moud' AIRFLOW_CTX_DAG_ID='dbt_orchestrator' AIRFLOW_CTX_TASK_ID='test_connection.dbt_test_connection' AIRFLOW_CTX_EXECUTION_DATE='2024-12-12T08:30:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-12T08:30:00+00:00'
[2024-12-12T08:45:01.252+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-12-12T08:45:01.262+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-12-12T08:45:01.262+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'dbt debug --profiles-dir /opt/***/dbt --project-dir /opt/***/dbt/ecommerce']
[2024-12-12T08:45:01.274+0000] {subprocess.py:86} INFO - Output:
[2024-12-12T08:45:03.041+0000] {subprocess.py:93} INFO - [0m08:45:03  Running with dbt=1.8.0
[2024-12-12T08:45:03.052+0000] {subprocess.py:93} INFO - [0m08:45:03  dbt version: 1.8.0
[2024-12-12T08:45:03.052+0000] {subprocess.py:93} INFO - [0m08:45:03  python version: 3.9.19
[2024-12-12T08:45:03.053+0000] {subprocess.py:93} INFO - [0m08:45:03  python path: /home/***/.local/bin/python
[2024-12-12T08:45:03.053+0000] {subprocess.py:93} INFO - [0m08:45:03  os info: Linux-6.8.0-49-generic-x86_64-with-glibc2.36
[2024-12-12T08:45:03.119+0000] {subprocess.py:93} INFO - [0m08:45:03  Using profiles dir at /opt/***/dbt
[2024-12-12T08:45:03.119+0000] {subprocess.py:93} INFO - [0m08:45:03  Using profiles.yml file at /opt/***/dbt/profiles.yml
[2024-12-12T08:45:03.120+0000] {subprocess.py:93} INFO - [0m08:45:03  Using dbt_project.yml file at /opt/***/dbt/ecommerce/dbt_project.yml
[2024-12-12T08:45:03.121+0000] {subprocess.py:93} INFO - [0m08:45:03  adapter type: postgres
[2024-12-12T08:45:03.121+0000] {subprocess.py:93} INFO - [0m08:45:03  adapter version: 1.8.0
[2024-12-12T08:45:03.202+0000] {subprocess.py:93} INFO - [0m08:45:03  [[33mWARNING[0m]: Deprecated functionality
[2024-12-12T08:45:03.202+0000] {subprocess.py:93} INFO - The `tests` config has been renamed to `data_tests`. Please see
[2024-12-12T08:45:03.202+0000] {subprocess.py:93} INFO - https://docs.getdbt.com/docs/build/data-tests#new-data_tests-syntax for more
[2024-12-12T08:45:03.202+0000] {subprocess.py:93} INFO - information.
[2024-12-12T08:45:03.248+0000] {subprocess.py:93} INFO - [0m08:45:03  Configuration:
[2024-12-12T08:45:03.249+0000] {subprocess.py:93} INFO - [0m08:45:03    profiles.yml file [[32mOK found and valid[0m]
[2024-12-12T08:45:03.249+0000] {subprocess.py:93} INFO - [0m08:45:03    dbt_project.yml file [[32mOK found and valid[0m]
[2024-12-12T08:45:03.250+0000] {subprocess.py:93} INFO - [0m08:45:03  Required dependencies:
[2024-12-12T08:45:03.260+0000] {subprocess.py:93} INFO - [0m08:45:03   - git [[32mOK found[0m]
[2024-12-12T08:45:03.261+0000] {subprocess.py:93} INFO - 
[2024-12-12T08:45:03.261+0000] {subprocess.py:93} INFO - [0m08:45:03  Connection:
[2024-12-12T08:45:03.261+0000] {subprocess.py:93} INFO - [0m08:45:03    host: 172.18.0.3
[2024-12-12T08:45:03.262+0000] {subprocess.py:93} INFO - [0m08:45:03    port: 5432
[2024-12-12T08:45:03.263+0000] {subprocess.py:93} INFO - [0m08:45:03    user: ***
[2024-12-12T08:45:03.263+0000] {subprocess.py:93} INFO - [0m08:45:03    database: ecommerce
[2024-12-12T08:45:03.264+0000] {subprocess.py:93} INFO - [0m08:45:03    schema: oltp
[2024-12-12T08:45:03.264+0000] {subprocess.py:93} INFO - [0m08:45:03    connect_timeout: 10
[2024-12-12T08:45:03.265+0000] {subprocess.py:93} INFO - [0m08:45:03    role: None
[2024-12-12T08:45:03.265+0000] {subprocess.py:93} INFO - [0m08:45:03    search_path: None
[2024-12-12T08:45:03.266+0000] {subprocess.py:93} INFO - [0m08:45:03    keepalives_idle: 0
[2024-12-12T08:45:03.266+0000] {subprocess.py:93} INFO - [0m08:45:03    sslmode: None
[2024-12-12T08:45:03.267+0000] {subprocess.py:93} INFO - [0m08:45:03    sslcert: None
[2024-12-12T08:45:03.267+0000] {subprocess.py:93} INFO - [0m08:45:03    sslkey: None
[2024-12-12T08:45:03.268+0000] {subprocess.py:93} INFO - [0m08:45:03    sslrootcert: None
[2024-12-12T08:45:03.268+0000] {subprocess.py:93} INFO - [0m08:45:03    application_name: dbt
[2024-12-12T08:45:03.269+0000] {subprocess.py:93} INFO - [0m08:45:03    retries: 1
[2024-12-12T08:45:03.270+0000] {subprocess.py:93} INFO - [0m08:45:03  Registered adapter: postgres=1.8.0
[2024-12-12T08:45:03.430+0000] {subprocess.py:93} INFO - [0m08:45:03    Connection test: [[32mOK connection ok[0m]
[2024-12-12T08:45:03.430+0000] {subprocess.py:93} INFO - 
[2024-12-12T08:45:03.431+0000] {subprocess.py:93} INFO - [0m08:45:03  [32mAll checks passed![0m
[2024-12-12T08:45:04.385+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-12-12T08:45:04.386+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-12-12T08:45:04.411+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dbt_orchestrator, task_id=test_connection.dbt_test_connection, run_id=scheduled__2024-12-12T08:30:00+00:00, execution_date=20241212T083000, start_date=20241212T084501, end_date=20241212T084504
[2024-12-12T08:45:04.468+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-12-12T08:45:04.512+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-12T08:45:04.514+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
